src/js/view/controlbar.js

```diff
@@ -361,6 +364,16 @@ define([
         onFullscreen : function(model, val) {
             utils.toggleClass(this.elements.fullscreen.element(), 'jw-off', val);
         },
+        checkControlSize : function(containerWidth) {
+            this.containerWidth = containerWidth || this.containerWidth;
+            if (this.containerWidth && this.containerWidth < this._minWidthForTimeDisplay) {
+                this.elements.elapsed.style.display = 'none';
+                this.elements.duration.style.display = 'none';
+            } else {
+                this.elements.elapsed.style.display = '';
+                this.elements.duration.style.display = '';
```

Comment:
Isn't this already done elsewhere or was that code removed?

Can we please make a css flag for this "" instead of setting it inline? it looks like this is still in the player and we could continue using it by toggling the flag https://github.com/jwplayer/jwplayer/blob/2d59678c4cc70fb8c98b05dfcea24e8a95d0dab0/src/css/flags/compact-player.less